CONTRIBUTING.md

```diff
@@ -0,0 +1,52 @@
+# Contributing to dxmentorship.com Rebuild
+
+Thank you for your interest in contributing to the dxmentorship.com rebuild project. We welcome your contributions to help make the project more efficient, user-friendly, and informative for mentees.
+
+## Types of Contributions
+
+You can contribute to this project in several ways, including but not limited to:
+
+- Reporting issues or bugs
+- Enhancing the website's functionality
+- Improving the user interface and user experience (UI/UX)
+- Adding content, such as details of previous graduates or curriculum information
+- Reviewing and improving code
+- Writing documentation
+
+## Getting Started
+
+Before you begin contributing, please follow these steps:
+
+1. Fork the repository to your GitHub account.
+2. Clone your forked repository to your local machine and add upstream remote:
+
+```bash
+git clone https://github.com/dxmentorship/dxmentorship.git
+cd dxmentorship
+git remote add upstream https://github.com/dxmentorship/dxmentorship.git
```

Comment:
The URL also would change to reflect the fork just like the one above.

Reply:
Thank you for the feedback 
I will make all necessary adjustments